_data/legal.yml

```diff
@@ -76,3 +76,13 @@ websites:
     regions:
       - gb
       - in
+
+  - name: Termly
+    url: https://termly.io
+    img: termly.svg
+    email_address: support@termly.io
+    twitter: Termly_io
+    facebook: termly.io
+    regions:
+      - gb
+      - us
```

Comment:
Why are the regions restricted to `gb` and `us`?

Reply:
I noticed a US/GB country switcher in the top right corner and assumed it meant that the site was only available in those regions, however it seems that the service is actually available globally, so I have removed the regions array.